wiki/Skinning/Compendium/Anime/en.md

```diff
@@ -0,0 +1,238 @@
+[o!s]: /wiki/shared/mode/osu.png "osu!standard"
+[o!t]: /wiki/shared/mode/taiko.png "osu!taiko"
+[o!c]: /wiki/shared/mode/catch.png "osu!catch"
+[o!m]: /wiki/shared/mode/mania.png "osu!mania"
+
+# Anime
+Compendium of all completed anime osu! skins.
+
+- ![][o!s]  - means the skin contains osu!standard elements.
```

Comment:
this list is not needed (tooltips serve as the explanation)

Reply:
There never really were any complains about them in previous versions. My biggest problem is that I can't just disable them, the output of the compendium is áutomated by a programm a friend wrote